### DISCUSSION TITLE
Can I use Flutter on the Windows side and Rust on the WSL side?

### DISCUSSION THREAD

**Discussion by satvikpendem** [fzyzcjy/flutter_rust_bridge#1081] [General]
I usually use Rust on the WSL portion of Windows, but if my Flutter app is on the native Windows side, will the two still be able to communicate?

**Comment by fzyzcjy** [comment]
Hmm flutter_rust_bridge invokes rust compilation during flutter compilation. So I am not sure, maybe possible, if you setup toolchain correctly?

**Reply by satvikpendem** [reply]
Yeah I just switched to using Rust and Flutter both on Windows now, it works well once you set everything up. Trying to call cargo from WSL into the Windows side is probably difficult.
